modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/conversion/swagger/v2_0/Swagger2Translator.java

```diff
@@ -407,16 +424,18 @@ private static void fillOperationParameters(Definition definition,
     }
 
     private static void fillOperationResponses(Definition definition,
-            Operation operation,
-            com.wordnik.swagger.models.Operation operationSwagger) {
-        for (Response response : operation.getResponses()) {
+            org.restlet.ext.apispark.internal.model.Operation operation,
+            Operation operationSwagger) {
+        for (org.restlet.ext.apispark.internal.model.Response response : operation.getResponses()) {
             /* Response -> Response */
-            com.wordnik.swagger.models.Response responseSwagger = new com.wordnik.swagger.models.Response();
+            Response responseSwagger = new Response();
 
             // may be null
-            String description = response.getDescription();
-            responseSwagger.setDescription((description != null) ? description
-                    : response.getCode() + " status response"); // required
+            String message = response.getMessage();
+            String autoGeneratedMessage = Status.isSuccess(response.getCode())
+                    ? "Success"
+                    : "Error " + response.getCode();
```

Comment:
"Error" is for errors only. Can you test if it's an error, and use "Status ..." otherwise?